### PR TITLE
add more information to readme

### PR DESCRIPTION
**Pull request by ameba23**
Explain the schema for each message type in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+  "type": "dark-crystal/root",
+  "version": "1",
+  "name": "directions to treasure",
+  "recps": ["@95WQAJ1XZju4YFpLib3JYdbx//BCtr5dq3bR9jPxYWs=.ed25519"]
+}
+```
+
+### `ritual`
+
+This message will also be published exactly once for each shared secret.  It will contain a reference to the `root` message, the required of shards to reconstruct the secret (quorum), which must be at least 2, the total number of shards, which must also be at least two, and a description of the software tool used to create the shards.  It will be a private message with exactly one recipient which should be the author of the message.
```
Comment (dan-mi-sun) [inline README.md]: the required **number** of shards to reconstruct

^^ missing word

Comment (dan-mi-sun) [reply README.md]: to be explicit the total number of shards required to _reconstruct_ the secret should always be less than or equal to the number of total shards.

Comment (ameba23) [reply README.md]: Ta dan.  I have added this to the readme.  although we are only validating that the number of shards >= 2.  So we should make it >= quorum.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+
+Example:
+
+```js
+{
+  "type": "dark-crystal/shard",
+  "version": "1",
+  "root": "%viiJnnnXjNkfCALivEZbrDe8UndkCCCNQ/CgBOWgJLw=.sha256",
+  "shard": "Yn3foQzIrckEh139UbZ2JYuQI9FSJ3lBEV7wcePeFc/Eeo0t9kfrNp+9+bZio76RTJOM7pVEo1AUJFFupGStwNHtXmcQ9msnvnvR1RW5qLxX3luNMe+m45jcDLDCwPU237TJFIqYbUbd/DeI3YFiFH+AMU8XAPTV9scukFMVSTDrr/Li6fI=.box",
+  "recps": ["@95WQAJ1XZju4YFpLib3JYdbx//BCtr5dq3bR9jPxYWs=.ed25519", "@95WQAJ1XZju4YFpLib3JYdbx//BCtr5dq3bR9jPxYWs=.ed25519"]
```
Comment (mixmix) [inline README.md]: only thing needs changing is the recps should be different!

I'm going to go ahead and merge this cos it looks good though

**Pull request merged**